src/api/controllers/RequestController.ts

```diff
@@ -38,8 +38,8 @@ export class RequestController {
     return { request: await this.requestService.deleteRequestById(params) };
   }
 
-  @Post('matches/id/:id/')
-  async getMatchesByRequestId(@Body() getMatchesRequest: GetMatchesRequest, @Params() params: UuidParam): Promise<GetPostsResponse> {
-    return { posts: await this.requestService.getMatchesByRequestId(params, getMatchesRequest) };
+  @Get('matches/id/:id/:time?/')
+  async getMatchesByRequestId(@Params() params: TimeParam): Promise<GetPostsResponse> {
```

Comment:
correct me if i'm wrong, but does this mean we're inputting the date through the url?

Reply:
Yeah we are